src/test/mochitest/browser_dbg-babel-stepping.js

```diff
@@ -104,23 +103,27 @@ add_task(async function() {
     ["stepOver", { line: 5, column: 2 }],
     ["stepOver", { line: 9, column: 2 }]
   ]);
+}
 
-  // The closure means it isn't actually possible to step into the for body,
-  // and Babel doesn't map the _loop() call, so we step past it automatically.
-  await breakpointSteps(
+// The closure means it isn't actually possible to step into the for body,
+// and Babel doesn't map the _loop() call, so we step past it automatically.
+function testStepOveForOfClosure(dbg) {
+  return breakpointSteps(
     dbg,
     "step-over-for-of-closure",
     { line: 6, column: 2 },
     [
       ["stepOver", { line: 8, column: 2 }],
-      ["stepOver", { line: 12, column: 2 }]
+      ["stepOver", { line: 8, column: 2 }]
```

Comment:
I changed this because the tests were failing locally. they were failing on CI too, but we weren't catching that...
https://travis-ci.org/devtools-html/debugger.html/branches

Reply:
We should expand this to be
```

      ["stepOver", { line: 8, column: 2 }],
      ["stepOver", { line: 8, column: 2 }],
      ["stepOver", { line: 8, column: 2 }],
      ["stepOver", { line: 12, column: 2 }]
```
so it still steps all the way to line 12.